examples/deploy-aws-lambda/app.js

```diff
@@ -9,11 +9,6 @@ const awsLambdaReceiver = new AwsLambdaReceiver({
 const app = new App({
   token: process.env.SLACK_BOT_TOKEN,
   receiver: awsLambdaReceiver,
-  // The `processBeforeResponse` option is required for all FaaS environments.
-  // It allows Bolt methods (e.g. `app.message`) to handle a Slack request
-  // before the Bolt framework responds to the request (e.g. `ack()`). This is
-  // important because FaaS immediately terminate handlers after the response.
-  processBeforeResponse: true
```

Comment:
Comment out this option is fine but as I mentioned above, we would like to keep the comment for OAuth flow use cases.

See also:
* https://github.com/slackapi/bolt-js/issues/1062
* https://github.com/slackapi/bolt-js/issues/1028
* https://github.com/slackapi/bolt-js/pull/1066/files#diff-2130e8227ce7a48a4f16812225568f8c3bca634968bd620e3c47fe9fa5f692c3R130